plugin.video.hardstyletv/addon.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.hardstyletv"
+	name="HardstyleTV"
+	version="1.0.0"
+	provider-name="alpadev">
+  <requires>
+    <import addon="xbmc.python" version="2.14.0"/>
+    <import addon="script.module.addon.common" version="2.0.0"/>
+    <import addon="plugin.video.youtube" version="5.0.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">Hardstyle livesets Youtube video Add-on</summary>
+    <description lang="en">HardstyleTV is an addon that streams Hardstyle livesets from the best festivals in the world. Qlimax, Defqon.1, Decibel, Hard Bass...we got all you need!</description>
+	<disclaimer lang="en">Some videos might be blocked in your country</disclaimer>
+    <platform>all</platform>
+    <source>https://github.com/aureliencousin/plugin.video.fullycharged</source>
+    <website>http://aurelien-cousin.com</website>
+	<language>en nl</language>
```

Comment:
this refers to media content language. Does the addon provides videos that are in english and dutch? If so, please ignore this comment. If not, please change it to the correct values.